qgrad/qgrad_qutip.py

```diff
@@ -377,3 +380,102 @@ def to_dm(state):
         )
 
     return out
+
+
+def make_rot(N, params, idx):
```

Comment:
Maybe we can make this a private function, so this should be `_make_rot`. It is a bit confusion to the user if this is a public method.